addons/im_livechat/static/src/public_models/public_livechat_global_notification_handler.js

```diff
@@ -1,7 +1,7 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
+import { attr, one } from '@mail/model/model_field';
```

Comment:
don't forget remove `attr` here (unused)

Reply:
@alexkuhn Oups, should be fine now :+1: